Sources/Core/JavaModelRenderer.swift

```diff
@@ -64,11 +64,7 @@ public struct JavaModelRenderer: JavaFileRenderer {
             } + ["this._bits = _bits;"]
         }
 
-        if params[.javaGeneratePackagePrivateSetters] == nil {
-            return [privateConstructor]
-        } else {
-            return [publicConstructor, privateConstructor]
-        }
+        return [publicConstructor, privateConstructor]
```

Comment:
I was trying to be stricter before and force the use of the Builder rather than exposing a public constructor (unless GeneratePackagePrivateSetters was enabled). There's no harm in exposing this constructor though - you still end up with a valid (empty) model.